dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -68,7 +68,7 @@ class ScriptLevelsControllerTest < ActionController::TestCase
     assert_response :success
     # toolbox blocks comes from project_level not real_level
     app_options = assigns(:level).blockly_options
-    assert_equal '<xml><toolbox/></xml>', app_options[:level]['toolbox_blocks']
+    assert_equal '<xml><toolbox/></xml>', app_options[:level]['toolbox']
```

Comment:
Nope, I made that mistake at first (fixed in abb3f90c777f5f31902b70a250551219d784efc7).  The camelization is overridden here: https://github.com/code-dot-org/code-dot-org/pull/2164/files#diff-7c558db0bac67f17752ae9f2a385c24fR158